Helper/Webhook.php

```diff
@@ -884,6 +884,12 @@ private function authorizePayment(Notification $notification)
         if ($isFullAmountAuthorized) {
             $this->setPrePaymentAuthorized();
             $this->prepareInvoice($notification);
+
+            // Set authorized amount in sales_order_payment
+            $orderAmountCurrency = $this->chargedCurrency->getOrderAmountCurrency($this->order, false);
+            $orderAmount = $orderAmountCurrency->getAmount();
+            $this->order->getPayment()->setAmountAuthorized($orderAmount);
```

Comment:
Can we move this change outside of this if condition? 
In that way, if this is a partial payment, the amount authorized will be updated on each partial payment webhook, instead of it being updated only once all the partial payments have been processed

Reply:
Sure, I moved it outside the full amount `if`.